src/components/Input.jsx

```diff
@@ -25,14 +23,6 @@ export default class Input extends React.Component {
     super(props);
   }
 
-  focus() {
-    ReactDOM.findDOMNode(this.input).focus();
```

Comment:
This functionality was disabled, please fix it.

Reply:
Done in https://github.com/kbss-cvut/record-manager-ui/pull/120/commits/7097e86f48ad28e185498efea7e6857e13addaf5